### ISSUE TITLE
No_Header in configuration file is not valid

### ISSUE THREAD

**Issue by Code-UV** [darold/ora2pg#1039]
When I use no_header in config file, it still print header in out file.

**Comment by darold** [comment]
Yes this is a command line option only `--no_header`, I will add the possibility to defined it in the configuration file.
